### PR TITLE
Intro which points to Biden Executive Order

### PR DESCRIPTION
**Pull request by ericvoit**
Per Henk's suggestion to change name and link to Biden's Executive Order

### CODE REVIEW

**Review by henkbirkholz — APPROVED** [review]
Captures the intent well!

File: draft-voit-rats-attestation-results.md [hunk 1]
```diff
@@ -75,6 +75,10 @@ informative:
     title: "802.1AR: Secure Device Identity"
     date: 2018-08-02   
   I-D.tschofenig-rats-psa-token: PSA
+  US Executive Order:
```
Comment (thomas-fossati) [inline draft-voit-rats-attestation-results.md:78]: I don't think Carsten's kdrfc likes empty spaces

```suggestion
  US-Executive-Order:
```

File: draft-voit-rats-attestation-results.md [hunk 2]
```diff
@@ -87,14 +91,19 @@ Additionally, where the Relying Party is interfacing with a heterogenous mix of
 
 # Introduction
 
-The Remote ATtestation procedureS (RATS) architecture {{-rats-arch}} defines conceptual messages conveyed between architectural subsystems to support trustworthiness appraisal.
-Within RATS, the Attestation Results conceptual message consists of "output generated by a Verifier, typically including information about an Attester, where the Verifier vouches for the validity of the results".
+The first paragraph of the May 2021 US Presidential Executive Order on Improving the Nation's Cybersecurity {{US Executive Order}} ends with the statement "the trust we place in our digital infrastructure should be proportional to how trustworthy and transparent that infrastructure is."
```
Comment (thomas-fossati) [inline draft-voit-rats-attestation-results.md:94]:
```suggestion
The first paragraph of the May 2021 US Presidential Executive Order on Improving the Nation's Cybersecurity {{US- Executive-Order}} ends with the statement "the trust we place in our digital infrastructure should be proportional to how trustworthy and transparent that infrastructure is."  
```

**Review by thomas-fossati — CHANGES_REQUESTED** [review]
Just a couple of typographical nits, which however are needed to make the source compile.

Also, at L73 the `802.1AR` label should not start with a number.  Maybe prepend `IEEE-` to it.

PS: I like the title change.

**Comment by ericvoit** [comment]
Ending comments with new version 01

**Pull request closed without merging**